contracts/DecentralandTemplate.sol

```diff
@@ -105,6 +105,9 @@ contract DecentralandTemplate is BaseTemplate, TokenCache {
         _createCustomAgentPermissions(_acl, _agent, _voting, _dclMultiSig);
         _createEvmScriptsRegistryPermissions(_acl, _voting, _voting);
         _createVotingPermissions(_acl, _voting, _voting, _tokenWrapper, _dclMultiSig);
+        
+        // HACK: create a random permission on TokenWrapper so it is detected as an app
+        _acl.createPermission(address(-1), _tokenWrapper, bytes32(-1), address(1));
```

Comment:
Is the manager intentionally address(*+*1)?

Also, consider that when we upgrade to Solidity 0.5.x, `bytes32(-1)` will fail with 'Explicit type conversion not allowed from "int_const -1" to "bytes32"', so I'd recommend using `bytes32(uint256(-1))` instead.

And also, there are is leftover whites pace a couple of lines above this one. Sorry, I've been hanging out with @sohkai

Reply:
> Is the manager intentionally address(+1)?

This is the [`BURN_ENTITY`](https://github.com/aragon/aragonOS/blob/next/contracts/acl/ACL.sol#L43).